src/Egulias/EmailValidator/EmailValidator.php

```diff
@@ -74,6 +112,26 @@ public function __construct()
         $this->parser = new EmailParser(new EmailLexer());
     }
 
+    /**
+     * Validates an email address against the following standards:
+     *
+     * RFC-5322: Internet Message Format
+     * RFC-5321: Simple Mail Transfer Protocol
+     * RFC-6530: Overview and Framework for Internationalized Email
+     * RFC-1123 section 2.1: Requirements for Internet Hosts -- Application and Support
+     * RFC-4291 section 2.2: IP Version 6 Addressing Architecture
+     *
```

Comment:
Okay, will do.  In my mind, I had been treating 6530 as the 'parent' to 6531 and 6532, but I suppose that's not really a fair way of looking at those RFCs!